### PR TITLE
Component/Page: Coming Soon PR

### PR DESCRIPTION
**Pull request by TylerEikenberg**
I made a page that just displays the CoTripper yellow and the words "Coming Soon". I made this for all the buttons on pages that don't lead anywhere, I figured we could have them link to this page instead of nothing. Totally up to you guys if you want to include this, I just thought it might be cool!

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,20 +1,47 @@
...
+);
 
 class App extends Component {
   render() {
     return (
-      <div className='App'>
+      <div className="App">
         <main>
           <Route path="/" exact component={SplashPage}></Route>
+          <Route path="/comingsoon" exact component={ComingSoonPage}></Route>
```
Comment (ZakkMan) [inline client/src/App.js]: Could we have the path be `coming-soon`?

Comment (ZakkMan) [reply client/src/App.js]: or `coming_soon`

Comment (TylerEikenberg) [reply client/src/App.js]: Lol, I'll fix it, thanks 👍

**Comment by TylerEikenberg** [comment]
@ZakkMan added your suggestions, let me know what ya think

**Pull request merged**